src/Database/Connection.php

```diff
@@ -254,11 +299,14 @@ public function getDisconnectRetry(): CommandRetry
     /**
      * Gets the driver instance.
      *
+     * @param string $role Connection role ('read' or 'write')
      * @return \Cake\Database\DriverInterface
      */
-    public function getDriver(): DriverInterface
+    public function getDriver(string $role = self::ROLE_WRITE): DriverInterface
```

Comment:
This could end up breaking userland implementations. We currently have `getDriver()` annotated only, but there could be a userland implementation of `ConnectionInterface`

Reply:
Derp. You're right as the method isn't on the interface.